src/components/custom/VoteCard.js

```diff
@@ -0,0 +1,53 @@
+import {Card} from "@/components/ui/card";
+import Image from "next/image";
+import {RejectedBadge} from "@/components/custom/RejectedBadge";
+import {PassedBadge} from "@/components/custom/PassedBadge";
+
+export function VoteCard({change, changeVotes, user}) {
```

Comment:
This should probably be called `ChangeCard` as it represents a change

Reply:
fixed